Management_System/BLL/DI/ServicesConfguration.cs

```diff
@@ -15,5 +16,7 @@ public static void AddBusinessLogicDependencies(this IServiceCollection services
         services.AddScoped<IBoardService, BoardService>();
         services.AddScoped<ITaskStatusService, TaskStatusService>();
         services.AddScoped<ILabelService, LabelService>();
+
+        services.AddAutoMapper(typeof(BoardProfile));
```

Comment:
May use Reflection to catch all mapping without need to add them in DI

Reply:
We can't use reflection to inject automapper